server/s3_get_object_action.cc

```diff
@@ -336,13 +336,72 @@ void S3GetObjectAction::read_object() {
   s3_log(S3_LOG_DEBUG, "", "%s Exit", __func__);
 }
 
+void S3GetObjectAction::check_outbuffer_and_mempool_stats(bool& bcontinue) {
+  s3_log(S3_LOG_DEBUG, stripped_request_id, "%s Entry\n", __func__);
+  bcontinue = true;
+  size_t motr_write_payload_size =
```

Comment:
You can have this outside and set to a member variable, so that it doesn't get invoked for each calls